### ISSUE TITLE
Presence of both: app title and app name is a bit confusing.

### ISSUE THREAD

**Issue by yakutovicha** [aiidalab/aiidalab#219]
Currently, we have app name and app title concepts.

App name is one word that is used to name the folder where the app is installed.  Also, under this name the app is registered in the aiidalab registry.

The title is a human-readable title that is only used to provide information for the users.

After some considerations, I would advocate for removing the "title" and making it a part of the description.

**Comment by csadorf** [comment]
Agreed, this would be similar to how PyPI displays information about a package. There is only one package name that is used as a unique identifier, everything else (e.g. a nicer representation of the package name) would be considered part of the description.
